src/proposals/20250807/SparkEthereum_20250807.t.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { IERC20 } from 'forge-std/interfaces/IERC20.sol';
+
+import { Ethereum } from 'spark-address-registry/Ethereum.sol';
+
+import { ChainIdUtils } from 'src/libraries/ChainId.sol';
+
+import { InterestStrategyValues, ReserveConfig } from '../../test-harness/ProtocolV3TestBase.sol';
+import { ICustomIRM, IRateSource } from '../../test-harness/SparkEthereumTests.sol';
+import { SparkLendContext }        from '../../test-harness/SparklendTests.sol';
+import { SparkTestBase }           from '../../test-harness/SparkTestBase.sol';
+
+contract SparkEthereum_20250807Test is SparkTestBase {
+
+    address internal constant PYUSD               = 0x6c3ea9036406852006290770BEdFcAbA0e23A0e8;
+    address internal constant PYUSD_PRICE_FEED    = 0x42a03F81dd8A1cEcD746dc262e4d1CD9fD39F777;
+    address internal constant USDS_DAI_IRM        = 0xD99f41B22BBb4af36ae452Bf0Cc3aA07ce91bD66;
+    address internal constant USDS_DAI_IRM_OLD    = 0xE15718d48E2C56b65aAB61f1607A5c096e9204f1;
+    address internal constant USDC_USDT_IRM_OLD   = 0x7F2fc6A7E3b3c658A84999b26ad2013C4Dc87061;
+    address internal constant USDC_USDT_PYUSD_IRM = 0xD3d3BcD8cC1D3d0676Da13F7Fc095497329EC683;
+
+    constructor() {
+        id = "20250807";
+    }
+
+    function setUp() public {
+        setupDomains("2025-07-29T15:43:00Z");
+
+        deployPayloads();
+
+        // chainData[ChainIdUtils.Ethereum()].payload = 0x41EdbF09cd2f272175c7fACB857B767859543D15;
+
+        // Deal the amount of pyUSD to Spark Proxy
+        deal(PYUSD, Ethereum.SPARK_PROXY, 1e6);
+    }
+
+    function test_ETHEREUM_sparkLend_collateralOnboarding() public onChain(ChainIdUtils.Ethereum()) {
+        SparkLendAssetOnboardingParams memory params = SparkLendAssetOnboardingParams({
+            // General
+            symbol:            'PYUSD',
+            tokenAddress:      PYUSD,
+            oracleAddress:     PYUSD_PRICE_FEED,
+            collateralEnabled: false,
+            // IRM Params
+            optimalUsageRatio:      0.95e27,
+            baseVariableBorrowRate: 0,
+            variableRateSlope1:     0.015e27,
```

Comment:
Move the ssr rate to the top here and make the `variableSlope1` the `ssrRate + spread`. Its kind of weird to have it dynamic like this but I think its clearer than calling the `variableSlope1Spread` the `variableSlope1`. I get why you did it though.

Reply:
okay